python/opendataloader-pdf-mcp/tests/test_convert_pdf.py

```
@@ -83,6 +91,103 @@ def test_quiet_is_always_true(self, input_pdf, tmp_path):
             kwargs = mock_convert.call_args[1]
             assert kwargs["quiet"] is True
 
+    def test_markdown_with_html_flag_is_forwarded(self, input_pdf, tmp_path):
+        """markdown_with_html=True should map to the convert() kwarg of the same name."""
+        fake_output = tmp_path / "lorem.md"
+        fake_output.write_text("mocked")
+
+        with patch("opendataloader_pdf_mcp.server.opendataloader_pdf.convert") as mock_convert, \
+             patch("opendataloader_pdf_mcp.server.tempfile.TemporaryDirectory") as mock_tmpdir:
+            mock_tmpdir.return_value.__enter__ = lambda self: str(tmp_path)
```

Ruff (0.15.15)

[warning] Unused lambda argument: `self`

(ARG005)